package.json

```diff
@@ -3,11 +3,15 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "start": "node node_modules/react-native/local-cli/cli.js start",
-    "start:logger": "LOGGER_ENABLED=true node node_modules/react-native/local-cli/cli.js start --reset-cache",
-    "link": "./node_modules/.bin/react-native link",
-    "ios": "./node_modules/.bin/react-native run-ios",
-    "android": "./node_modules/.bin/react-native run-android",
+    "start": "react-native start",
+    "start:logger": "LOGGER_ENABLED=true react-native start --reset-cache",
+    "start:android": "concurrently -r 'react-native start' 'react-native run-android'",
+    "start:android:logger": "LOGGER_ENABLED=true concurrently -r 'react-native start' 'react-native run-android'",
+    "start:ios": "concurrently -r 'react-native start' 'react-native run-ios'",
+    "start:ios:logger": "LOGGER_ENABLED=true concurrently -r 'react-native start' 'react-native run-ios'",
```

Comment:
Do these need `react-native start --reset-cache`? That's what the `start:logger` script has, not sure if it's important.